docsite/rst/intro_configuration.rst

```diff
@@ -160,7 +160,10 @@ stdout_callback
 
 .. versionadded:: 2.0
 
-This setting allows you to override the default stdout callback for ansible-playbook.
+This setting allows you to override the default stdout callback for
+ansible-playbook::
```

Comment:
no need for this space change, we don't have any users stuck with an 80 column display